user_documentation/configuration.tex

```diff
@@ -56,22 +62,48 @@ \section{External Macro Libraries and COPY Members}
 	\label{lst:grps}
 \end{listing}
 
-There is also the option \TT{alwaysRecognize} which takes an array of wildcards. It allows you to configure two things:
+An example configuration can be seen in \cref{lst:ex1} and \cref{lst:grps}. It describes two processor groups \TT{GROUP1}, \TT{GROUP2} and their respective mappings to programs \TT{source} and \TT{asm\_prg}. If you have the two configuration files configured as above and invoke the MAC1 macro from \TT{source}, the folder \TT{ASMMAC/} in the current workspace is searched for a file named exactly MAC1. If that file is not found, the folder \TT{C:/SYS.ASMMAC} is searched. If that search is unsuccessful too, an error displays that the macro does not exist.
+
+Note that the macro \TT{MAC1} is searched in directories in order as they are listed in the configuration. 
+
+To add a possibility to specify a custom extension for HLASM files in the workspace, {pgm\_conf.json} has an optional field \TT{alwaysRecognize}. It takes an array of wildcards and provides the following functionality:
 \begin{itemize}
-	\item All files matching these wildcards will always be recognized as HLASM files. 
+	\item All files matching these wildcards will always be recognized as HLASM files. Hence, the HLASM plugin will be automatically activated on all the matched files.
```

Comment:
Not true really... On bezi stale :D

Reply:
chcel som tym viac zdoraznit, ze budes na tom file moct pouzivat funkcionalitu nasho celeho pluginu. ako by som to mal preformulovat?

Reply:
Jasne, chapem. Mozno nahradit activated za running? Alebo "the HLASM plugin will be automatically processing all matched files"?